src/api_wrapper.py

```diff
@@ -13,10 +13,19 @@
 }
 
 
-def fetch_api(plz: int, birthdate_timestamp: int = None, max_retries: int = 10, sleep_after_error: int = 30, sleep_after_shadowban: int = 300) -> any:
-    url = f"https://www.impfportal-niedersachsen.de/portal/rest/appointments/findVaccinationCenterListFree/{plz}"
+def fetch_api(
+        plz: int,
+        birthdate_timestamp: int = None,
+        group_size: int = None,
+        max_retries: int = 10,
+        sleep_after_error: int = 30,
+        sleep_after_shadowban: int = 300
+    ) -> any:
+    url = f"https://www.impfportal-niedersachsen.de/portal/rest/appointments/findVaccinationCenterListFree/{plz}?stiko="
     if birthdate_timestamp is not None:
```

Comment:
could be shorten to if birthdate_timestamp:

Reply:
Done (ff13509)